<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox::indicator:checked {
image: url(:/image/check_on.png);
}

QGroupBox::indicator:unchecked {
image: url(:/image/check_off.png);
}
QGroupBox::indicator:checked:!enabled {
image: url(:/image/disable_check_on.png);
}

QGroupBox::indicator:unchecked:!enabled {
image: url(:/image/disable_check_off.png);
}

QGroupBox::indicator {
width: 17px;
height: 17px;
}

/*tabwidget*/

QTabWidget{
background:transparent;

}
QTabWidget QTabBar::tab{
height:30px;
background:transparent;
}

QTabWidget QTabBar::tab:selected{
background:transparent;
color:purple;
font:bold;
}
QTabWidget::pane { /* The tab widget frame */
border-top: 1px solid gray;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>退出</string>
        </property>
        <property name="icon">
         <iconset resource="peripheral.qrc">
          <normaloff>:/image/close.png</normaloff>:/image/close.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>小票打印</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>150</y>
          <width>160</width>
          <height>100</height>
         </rect>
        </property>
        <property name="text">
         <string>测试</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>481</width>
          <height>291</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_5" rowstretch="0,0,0">
         <item row="0" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>打印机类型</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>串口</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>端口</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_2"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>波特率</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBox_3"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="title">
            <string>USB</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>端口</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_4"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>网口</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>ip</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>port</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>VIDPID</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>端口</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_3"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLineEdit" name="lineEdit_hex_path">
        <property name="geometry">
         <rect>
          <x>63</x>
          <y>5</y>
          <width>146</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>/dev/usb/lp0</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_hex_test">
        <property name="geometry">
         <rect>
          <x>456</x>
          <y>5</y>
          <width>85</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>测试</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_hex_content">
        <property name="geometry">
         <rect>
          <x>293</x>
          <y>5</y>
          <width>146</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>31 0a</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>5</y>
          <width>45</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>端口号</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>227</x>
          <y>5</y>
          <width>60</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>十六进制</string>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>621</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>计价秤</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>80</y>
          <width>160</width>
          <height>100</height>
         </rect>
        </property>
        <property name="text">
         <string>测试</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>351</width>
          <height>256</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>计价秤类型</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_5"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>端口</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox_9"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>波特率</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox_6"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>回读数据</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_4"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>超时(ms)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit_pricescale_timeout">
           <property name="text">
            <string>100</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>回读成功率</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit_countpricerate"/>
         </item>
         <item row="6" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>客显</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>80</y>
          <width>160</width>
          <height>100</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>测试</string>
        </property>
        <property name="outer_stylesheet" stdset="0">
         <string>pushbutton</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>331</width>
          <height>171</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label_77">
           <property name="text">
            <string>客显类型:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_7"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_78">
           <property name="text">
            <string>端口:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox_8"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>显示内容</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_6">
           <property name="text">
            <string>-12.6</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>客显启动内容:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_5">
           <property name="text">
            <string>welcome</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>读卡器</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>100</y>
          <width>160</width>
          <height>100</height>
         </rect>
        </property>
        <property name="text">
         <string>蜂鸣</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>条码打印机</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>100</y>
          <width>160</width>
          <height>100</height>
         </rect>
        </property>
        <property name="text">
         <string>条码打印机</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="peripheral.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>685</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>532</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
